### ISSUE TITLE
The test WCAG-T28 reports too many failures due to not actually testing for important

### ISSUE THREAD

**Issue by tolnem** [qualweb/wcag-techniques#4]
I thought I already created an issue for this, but I can't find it anywhere. If this is a duplicate, I apologise.

It looks like the latest commit to the WCAG-T28 test actually removed the code that checks for important in the font size definition.

I've created a small test page at https://test.symbiotisk.dk/WCAG-T28.html that proves the issue. It doesn't use important anywhere, but still fails the test.
